reproducibility_FAQs.md

```diff
@@ -0,0 +1,253 @@
+-   **Use formulas and macros**  where appropriate
+-   **Document all transformations clearly**  — either in a separate log file or within the README
+-   **Explain how outputs were created from Excel files**, especially if they appear in the manuscript
+    
+
+
+ [See our Excel guidelines](https://worldbank.github.io/wb-reproducible-research-repository/resources/excel/Excel_Guidelines.html)  for recommended practices and our [presentation on how to produce tables and plots in Stata and R](https://osf.io/ezmsb) for quick instructions on how to build reproducible outputs.
+
+## Data-Related Questions
+
+### What is original data?
+Original data refers to a citable dataset—one that is documented, publicly accessible or archived in a World Bank internal catalog—and serves as the starting point for reproducibility.  A few options include:
+-   Unmodified raw data: As originally received, from surveys, administrative systems, or fieldwork, with no changes beyond the removal of personally identifiable information(PII).  
+-   Published datasets: Datasets from trusted sources like the Microdata Library, LSMS, DHS, etc.  
+-   Official statistics & indicators: Data from sources like the World Development Indicators, national statistics portals, or IMF/WEO
+-   Data as it was received from another WB team or an external counterpart, except for the removal of personally identifiable information (see the question below on documenting the data sources for more information).
+
+### Can you do a reproducibility verification if the research relies on confidential data?
+
+Reproducibility verification is still possible when data access is restricted. We use several strategies depending on the nature of the restriction:
+
+-   **Data access for verification only**
+    -   The reproducibility team may request temporary access directly from the data provider.
+    -   If needed, we can sign a non-disclosure agreement (NDA) to access the data solely for verification purposes.        
+-   **Virtual verification**
+    -   The author runs the full package in a prepared environment in their computer (e.g., via screen sharing or secure remote session).
+    -   The reviewer observes the process and receives the resulting outputs for comparison.
+-   **Synthetic data approach**
+    -   Authors share synthetic datasets that mimic the structure of the real data.
+    -   The reviewer verifies reproducibility using the actual data, while the public package includes only the synthetic version.
+    
+These approaches help ensure transparency and reproducibility without compromising data confidentiality.
+
+
+### How do I document the datasets used in the reproducibility package?
+
+All datasets used in the package must be documented in the **Data Availability Statement**. This should include the following information based on the source of the data:
+
+#### 🔹 If using data generated by others:
+- Provide a full data citation or DOI, or include:
```

Comment:
link to the social science data editors guidance on creating a data citation. emphasize that data citations are expected.